examples/rosettacode/24_game.nit

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env nit
+#
+# This file is part of NIT ( http://www.nitlanguage.org ).
+# This program is public domain
+
+# Task: 24 game
+# SEE: <http://rosettacode.org/wiki/24_game>
+
+# Export `operands` and `numbers` from string `operation`
+fun exportation(operation: String, numbers: Array[Int], operands: Array[Char]) do
+	var previous_char: nullable Char = null
+	var number: nullable Int = null
+	var negative = false
+
+	for i in operation.length.times do
+		var current_char = operation[i]
+		var current_int = current_char.to_i
+
+		if (previous_char == null or "-+/*".has(previous_char)) and current_char == '-' then
```

Comment:
`"-+/*".has(previous_char)` is so often used, why not move it to a method?
~~~
redef universal Char
    fun is_op: Bool do return "-+/*".has(self)
end
~~~